<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepresentationUI</class>
 <widget class="QMainWindow" name="RepresentationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AIM Representation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="OIMImportWidget" name="oimImportWidget">
       <attribute name="title">
        <string>OIM Data Import</string>
       </attribute>
      </widget>
      <widget class="ReconstructionWidget" name="reconstructionWidget">
       <attribute name="title">
        <string>Reconstruction</string>
       </attribute>
      </widget>
      <widget class="GrainGeneratorWidget" name="grainGeneratorWidget">
       <attribute name="title">
        <string>Synthentic Builder</string>
       </attribute>
      </widget>
      <widget class="SurfaceMeshingWidget" name="surfaceMeshingWidget">
       <attribute name="title">
        <string>Surface Meshing</string>
       </attribute>
      </widget>
      <widget class="VolumeMeshingWidget" name="volumeMeshingWidget">
       <attribute name="title">
        <string>Volume Meshing</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_RecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actionSomething"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="menu_RecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSomething">
   <property name="text">
    <string>Something</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReconstructionWidget</class>
   <extends>QFrame</extends>
   <header location="global">ReconstructionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OIMImportWidget</class>
   <extends>QFrame</extends>
   <header location="global">OIMImportWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GrainGeneratorWidget</class>
   <extends>QFrame</extends>
   <header location="global">GrainGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SurfaceMeshingWidget</class>
   <extends>QFrame</extends>
   <header location="global">SurfaceMeshingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeMeshingWidget</class>
   <extends>QFrame</extends>
   <header location="global">VolumeMeshingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
